### ISSUE TITLE
Add support of IPv6 address for virtual compute nodes

### ISSUE THREAD

**Issue by dimm0** [sdsc/nucleus-roll#16]

**Comment by tcooper** [comment]
Support for IPv6 for public IPs for compute nodes requires support from the networking team as well as policy from the security team.

**Comment by dimm0** [comment]
Waiting for Tom to enable IPV6 on gateway switch

**Comment by dimm0** [comment]
Close this as won't fix?..

**Comment by tcooper** [comment]
No... we need to keep this on hold. Perhaps move it to a new milestone.

**Comment by dimm0** [comment]
I thought no money - no honey... No routers supporting v6 - no v6?

**Comment by tcooper** [comment]
No support in *current* software.
